cartography/intel/gcp/apigateway.py

```diff
@@ -367,7 +367,7 @@ def get_api_gateway_policy_entities(apigateway: Resource, gateway: Dict, project
         iam_policy = apigateway.projects().locations().gateways().getIamPolicy(
             resource=gateway.get('name')).execute()
         bindings = iam_policy.get('bindings', [])
-        entity_list, public_access = iam.transform_bindings(bindings, project_id)
+        entity_list, public_access = transform_iam_bindings(bindings, project_id)
```

Comment:
I don't see this output used anywhere. `get_api_gateway_policy_entities` method is not used at all. let's remove this method if not used?

Reply:
It is now used in the `get_gateways` method